src/main/java/edu/harvard/iq/dataverse/DatasetPage.java

```diff
@@ -3025,6 +3010,11 @@ public boolean isSelectAllFiles() {
     public void setSelectAllFiles(boolean selectAllFiles) {
         this.selectAllFiles = selectAllFiles;
     }
+    
+    public void handleFileSelectClick(){
```

Comment:
is this used anywhere? It's new, but I don't see any other reference.

Reply:
Removed. I think I put it in in anticipation of giving immediate feedback when the files selected were too large for zip.